content/jobs/_index.md

```diff
@@ -55,17 +54,107 @@ Interested candidates should email an application to `info@openforcefield.org` w
+found [here](http://openforcefield.org/publications).
+
+The [Open Force Field Initiative](http://openforcefield.org) is a multi-investigator collaboration 
+that aims to build new, quantitatively accurate molecular mechanics force fields supported by a 
+modern software infrastructure, built on principles of open source, open data, and open science. 
+More about their scientific goals and plans can be found in the Open Force Field Initiative 
+[Executive Summary](https://openforcefield.org/science/downloads/roadmap/open-forcefield-summary.pdf) or 
+[Roadmap](https://openforcefield.org/science/downloads/roadmap/open-forcefield-plan.pdf).
+
+#### Software Scientists at Open Force Field
+
+Software Scientists on our team are high-level research engineers (often with a Ph.D.) with strong science, 
+programming, and communication skills who can simultaneously tackle long-term development efforts and 
+perform ongoing organizational duties. The desired qualifications for this role include significant 
+experience with scientific software development (specifically Python), testing, documentation, and 
+deployment, in addition to the “desired qualifications” for a postdoc. The software scientist is expected 
+to perform duties ranging from working asynchronously and independently to implement major features, to 
+closely collaborating with a team to produce a deliverable by a deadline. The Software Scientist’s effort 
+will be divided approximately 50% to architecture, coding, and science, 25% to devops and user support, 
+and 25% to management and programming guidance. Some specific examples of responsibilities include:
+
+- Publishing [software releases and datasets with DOIs](https://zenodo.org/record/3541134#.Xd1xbb-IbOQ), 
+  with clear documentation that references scholarly works and theory when applicable 
+- Becoming the primary developer/maintainer for a core Python toolkit 
+- Being a technical mentor to grad students and postdocs, performing code reviews, helping with 
+  architecture decisions, and sharing modern software best practices 
+- Working one-on-one with industry partners to install and make workflows using our software 
+- Broadening our engagement with the open-source chemistry community 
+- Leading regular science subproject meetings and writing 
+  [blog posts](https://openforcefield.org/science/updates/propertycalculator-meeting-2019-10-17/) to 
+  summarize progress 
+- Representing Open Force Field at conferences and workshops 
+- Traveling to [project labs](https://openforcefield.org/members/) as needed for focused in-person work 
+- Debugging CI builds, dealing with conda packaging issues, and helping shepherd the eventual migration 
+  of omnia packages to conda-forge 
+
+Writing papers is _not_ a major focus of this position.
+
+The position includes up to 25% travel, and the software scientist will be assigned a PI that they will 
```

Comment:
Does "25% travel" mean that you will be working from somewhere besides your primary location 25% of the year? That's a lot!

Reply:
Figured it was safest to cover our bases with a "worst case" statement, but many people have mentioned this now. I've dropped it to 15% (which is still 7 weeks in a year)